recipe/meta.yaml

```diff
@@ -58,8 +62,8 @@ outputs:
         - numpy                                  # [build_platform != target_platform]
         - libtool                                # [osx and arm64]
         - libprotobuf
-        - grpc-cpp
```

Comment:
Any reason why this was deleted? This breaks the `osx-arm64` build.

Reply:
Nope that was unintentional, sorry. Let's revert that commit (or rebase it out)